doc/slha_input.rst

```diff
@@ -638,9 +638,13 @@ physical input parameters, which are not contained in a SLHA-compliant
 ======= ====================================== ============================== ==================
   0      alpha_em(0) in the Thompson limit      any positive double            1./137.035999074
   1      SM Higgs pole mass                     any positive double            125.09
+  2      Planck constant, reduced               any positive double            6.582119569e-25
```

Comment:
Do we really need an option to change hbar? This is probably an overkill. Maybe we could hardcode it in `src/wrapper.hpp` where for whatever reason we put all the constants? What do you think?

Reply:
Numerical constants live  in ``src/wrappers.hpp``. This one is a physical one. I think that this might be a little bit confusing to put it there. But I think, in general, that this might be a correct solution to put it somewhere else, because it is hard to foresee, someone tinkering with its value.